### DISCUSSION TITLE
My component with wire:ignore bugs when the user goes on 'previous page' after a wire:navigate

### DISCUSSION THREAD

**Discussion by joffreypersia** [livewire/livewire#8353] [1. Help]
Hi,

I have a page with a lot of elements displayed in table with a filter bar. 
In the filter bar, I have custom coded select component that receives an array of options. Because we use Livewire, we put a wire:ignore on root select tag to prevent the component to reload when the user use the select to filter the table. If we remove the wire:ignore, the component select is reload with the table and we loose the state of the select. 

We switched our app to wire:navigate for the SPA feel, but I noticed that when the user use the previous page on its browser, it makes errors on the components selects (that have the wire:ignore). When the page is loaded after a previous page, my component now have 2 times the number of options, with nulled options that causes problems on my app. 

Someone already managed wire:ignore and previous page load with wire:navigate ? 

![CleanShot 2024-04-26 at 16 31 09@2x](https://github.com/livewire/livewire/assets/94628273/876ffc3e-7264-4937-ad71-7af820e80751)
![CleanShot 2024-04-26 at 16 33 10@2x](https://github.com/livewire/livewire/assets/94628273/00bd7093-c342-47b5-ba28-7963dcd50ce9)

Thank you !

**Comment by joshhanley** [comment]
We're in the process of investigating all wire:navigate issues and I believe this is possible using `@persist` so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
